deployments/helm/nvidia-device-plugin/templates/daemonset-device-plugin.yml

```diff
@@ -57,6 +57,7 @@ spec:
       {{- end }}
       {{- if $options.hasConfigMap }}
       shareProcessNamespace: true
+      hostPID: {{ .Values.mps.hostPID }}
```

Comment:
Should we not be making this change to the MPS control daemon daemonset instead?

Reply:
Whoops :). Thanks for catching that! But what are your thoughts on applying this setting consistently across the device plugin and MPS daemon?

Reply:
I think this goes against the work that we're doing to limit the permissions that we expose in our components. The device plugin should not require host IPC.